cli/installer/docker_compose.go

```diff
@@ -166,7 +166,7 @@ func getDockerComposeFileContents(ui UI, config configuration) []byte {
 		ui.Exit(fmt.Sprintf("cannot configure tracetest container: %s", err.Error()))
 	}
 
-	if config.Bool("tracetest.collector.install") {
+	if config.Bool("tracetest.collector.install") && config.Bool("tracetest.backend.install") {
 		if err := fixOtelCollectorContainer(config, project); err != nil {
```

Comment:
This relies on Jaeger. So it doesn't have to be executed if jaeger is not being installed